status_of_code.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+class StatusOfCode(Enum):
+    SchemaNotFound = "The json file you entered is not found"
+    ErrorLoadingJsonFile = "The data in the json file could not be loaded"
```

Comment:
Enum is an integer not a message.

Reply:
Ok I just wanted every status to be attached to a certain message but I will think of another way to do so